addons/website/static/src/snippets/s_rating/options.js

```diff
@@ -0,0 +1,160 @@
+odoo.define('website.s_rating_options', function (require) {
+'use strict';
+
+const weWidgets = require('wysiwyg.widgets');
+const options = require('web_editor.snippets.options');
+
+options.registry.Rating = options.Class.extend({
+    /**
+     * @override
+     */
+    start: function () {
+        this.faClassActiveCustomIcons = '';
+        this.faClassInactiveCustomIcons = '';
+        this.iconType = this.$target[0].dataset.icon;
+        this.nbActiveIcons = this.$target.find('.s_rating_active_icons > i').length;
+        this.nbTotalIcons = this.$target.find('.s_rating_icons i').length;
+        if (this.iconType === 'custom') {
+            this.faClassActiveCustomIcons = this.$target[0].dataset.activeCustomIcon;
+            this.faClassInactiveCustomIcons = this.$target[0].dataset.inactiveCustomIcon;
```

Comment:
Instead of defining those to an empty string just above, can't you do:
```javascript
this.faClassMachin = this.$target[0].dataset.activeCustomIcon || '';
```
?